From 051fad748c27787f69541e62fac3c7716cd5d526 Mon Sep 17 00:00:00 2001
From: Pete Robbins <robbinspg@gmail.com>
Date: Tue, 5 Jan 2016 13:10:21 -0800
Subject: [PATCH 053/201] [SPARK-12647][SQL] Fix
 o.a.s.sqlexecution.ExchangeCoordinatorSuite.determining
 the number of reducers: aggregate operator

change expected partition sizes

Author: Pete Robbins <robbinspg@gmail.com>

Closes #10599 from robbinspg/branch-1.6.

(cherry picked from commit 5afa62b20090e763ba10d9939ec214a11466087b)
---
 .../sql/execution/ExchangeCoordinatorSuite.scala   |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sql/core/src/test/scala/org/apache/spark/sql/execution/ExchangeCoordinatorSuite.scala b/sql/core/src/test/scala/org/apache/spark/sql/execution/ExchangeCoordinatorSuite.scala
index 180050b..59bb354 100644
--- a/sql/core/src/test/scala/org/apache/spark/sql/execution/ExchangeCoordinatorSuite.scala
+++ b/sql/core/src/test/scala/org/apache/spark/sql/execution/ExchangeCoordinatorSuite.scala
@@ -318,7 +318,7 @@ class ExchangeCoordinatorSuite extends SparkFunSuite with BeforeAndAfterAll {
         }
       }
 
-      withSQLContext(test, 1536, minNumPostShufflePartitions)
+      withSQLContext(test, 1700, minNumPostShufflePartitions)
     }
 
     test(s"determining the number of reducers: join operator$testNameNote") {
-- 
1.7.9.5

